utils/helpers.js

```diff
@@ -1,12 +1,30 @@
+import axios from 'axios';
 /**
- * Parses ipfs:// links and IPFS hashes to URLs.
+ * Parses ipfs:// and ar:// links and IPFS hashes to URLs.
  * @param {String} uri The ipfs:// link or IPFS hash.
  * @returns A formatted URL to the IPFS resource.
  */
-const parseIPFS = uri => {
-    return uri.substring(0,4) === "ipfs" ? 
-    `https://ipfs.zesty.market/ipfs/${uri.substring(7)}` :
-    `https://ipfs.zesty.market/ipfs/${uri}`;
+const parseProtocol = uri => {
+  if (uri.substring(0,4) === "ipfs") {
+    return `https://ipfs.zesty.market/ipfs/${uri.substring(7)}`;
+  } else if (uri.substring(0,4) === "http") {
```

Comment:
This and line 12 (`uri.substring(0,5)`) are equivalent if you think about it.

Reply:
Yes I agree, I was reasoning it as some kind of switch statement. Could do it as an `||`. I don't think performance is dramatically different. Mainly a stylistic issue, feel free to change if desired.